include/ignition/gazebo/Types.hh

```diff
@@ -87,6 +89,8 @@ namespace ignition
 
     /// \brief A key that uniquely identifies, at the global scope, a component
     /// instance
+    // TODO(anyone) remove this in ign-gazebo7. This is no longer used in
+    // ign-gazebo6
     using ComponentKey = std::pair<ComponentTypeId, ComponentId>;
```

Comment:
I left a TODO note to remove this because I don't think it's possible to mark a type alias as deprecated, right?